lib/rucio/core/rule.py

```diff
@@ -1424,17 +1425,18 @@ def reduce_rule(rule_id, copies, exclude_expression=None, session=None):
 
 
 @transactional_session
-def move_rule(rule_id, rse_expression, activity=None, source_replica_expression=None, session=None):
+def move_rule(rule_id: str, rse_expression: str, override: Dict[str, Any] = None, session=None):
```

Comment:
I believe `Optional[Dict[str, Any]]` is more correct here.

Reply:
Yes, I am not being able to see the wood for trees...